docker/Dockerfile.marimo

```
@@ -168,3 +171,38 @@ RUN \
+
+FROM base AS micromamba
+
+COPY --from=micromamba-installer /mnt/micromamba /usr/local/bin/micromamba
+ENV MAMBA_ROOT_PREFIX="/home/${USER}/.local/share/micromamba"
+ENV MAMBA_EXE="/usr/local/bin/micromamba"
+RUN <<EOF
+mkdir -p /home/${USER}/.local/share/micromamba /home/${USER}/.local/bin
+ln -s /usr/local/bin/micromamba /home/${USER}/.local/bin/mamba
+ln -s /usr/local/bin/micromamba /home/${USER}/.local/bin/conda
+EOF
+ENV PATH="/home/${USER}/.local/bin:${PATH}"
```

Hadolint (2.15.1)

[error] unexpected 'E'
expecting a new line followed by the next instruction

(DL1000)